config/examples/Creality/CR-10 V3/Configuration.h

```diff
@@ -1380,12 +1380,12 @@
 // @section machine
 
 // The size of the printable area
-#define X_BED_SIZE 310 // Nozzle is at X0 when homed
-#define Y_BED_SIZE 310 // Nozzle is at Y0 when homed
+#define X_BED_SIZE 300 // Nozzle is at X0 when homed
+#define Y_BED_SIZE 300 // Nozzle is at Y0 when homed
 
 // Travel limits (mm) after homing, corresponding to endstop positions.
-#define X_MIN_POS 0
-#define Y_MIN_POS 0
+#define X_MIN_POS -5 // -5 instead of 0 to center the print area (CR-10 v3)
+#define Y_MIN_POS -5 // -5 instead of 0 to center the print area (CR-10 v3)
```

Comment:
To center the print area

Reply:
It should be based on having 0, 0 close to the front-left corner of the printable area, then adjust the bed size to achieve better centering.